aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentDetailsUtilTest.java

```diff
@@ -0,0 +1,70 @@
+package design.aem.utils.components;
+
+import com.day.cq.wcm.api.Page;
+import com.day.cq.wcm.api.PageManager;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ValueMap;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static design.aem.utils.components.CommonUtil.PN_REDIRECT_TARGET;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ComponentDetailsUtilTest {
+
+    @Mock
+    Page page, currentPage;
+
+    @Mock
+    ResourceResolver resolver;
+
+    @Mock
+    Resource contentResource;
+
+    @Mock
+    ValueMap valueMap;
+
+    @Mock
+    PageManager pageManager;
+
+    @Before
+    public void before() {
+        initMocks(this);
+    }
+
+    @Test
+    public void testClass() {
+        ComponentDetailsUtil test = new ComponentDetailsUtil();
```

Comment:
This is redundant. You would get build/compilation errors if this class didn't exist

Reply:
Removed now.